aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsEndpointDecorator.kt

```diff
@@ -37,7 +37,7 @@ import software.amazon.smithy.rust.codegen.core.util.letIf
 
 class AwsEndpointDecorator : ClientCodegenDecorator {
     override val name: String = "AwsEndpoint"
-    override val order: Byte = -100
+    override val order: Byte = 100
```

Comment:
Would you mind adding a comment that says something like "ensure this runs AFTER other decorators" so that these are more understandable

Reply:
Cause I still always forget lol